static/bidder-info/nobid.yaml

```diff
@@ -0,0 +1,11 @@
+maintainer:
+  email: "reda.guermas@nobid.io"
```

Comment:
Are you able to use a more general mailbox in case someone else assumes maintenance responsibility later on?

Reply:
Done

Reply:
Thank you.